cmd/pd-server/main.go

```diff
@@ -50,6 +52,13 @@ func main() {
 		log.Fatalf("parse cmd flags error: %s\n", err)
 	}
 
+	dataDir, err := filepath.Abs(cfg.DataDir)
+	logFile, err := filepath.Abs(cfg.Log.File.Filename)
+	rel, err := filepath.Rel(dataDir, filepath.Dir(logFile))
+	if !strings.HasPrefix(rel, "..") {
```

Comment:
sometimes user may not use `..` here.

Reply:
Rel() gets relative path based on absolute path, it has nothing to do with whether user use ".." or not